src/internal/observable/bindNodeCallback.ts

```diff
@@ -5,148 +5,18 @@ import { bindCallbackInternals } from './bindCallbackInternals';
 
 /** @deprecated resultSelector is deprecated, pipe to map instead */
 export function bindNodeCallback(
-  callbackFunc: (...args: any[]) => any,
-  resultSelector: (...args: any[]) => any,
+  callbackFunc: (...args: any[]) => void,
+  resultSelector: (...args: any[]) => void,
```

Comment:
nitpick: `resultSelector` should return `any`.

Reply:
Good catch